common-data/issue-choices.json

```diff
@@ -1,5 +1,5 @@
 [
-    ["HOME__COVID_SANITATION_REQUIRED", "COVID-19 Sanitation Required"],
+    ["HOME__COVID_SANITATION_REQUIRED", "Needs cleaning due to COVID-19"],
```

Comment:
Cool, so one really annoying thing about changing this file is that it ultimately winds up being part of a Django model field's valid choices, which means (ridiculously) that we need to create a database migration for it.  Can you run `python manage.py makemigrations` and add the migration it creates?

(After googling I actually found what appears to be a clever way to work around this annoyance and filed it as #1098, which we can address later!)

Reply:
Ahh ok, yeah good to know that even small changes like this still require a db migration.